src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -270,10 +272,33 @@ public static Maps instance() {
         if (actual.isEmpty() && values.length == 0) return;
 
         Set<V> expected = HashSet.of(values);
-        Set<V> notFound = expected.filter(notPresentIn(actual.values()));
+        Set<V> notFound = expected.filter(valueNotPresentIn(actual.values()));
         if (isNotEmpty(notFound)) throw failures.failure(info, shouldContainValues(actual, notFound.toJavaSet()));
     }
 
+    /**
+     * Verifies that the actual map does not contain the given values.
+     *
+     * @param <K>    key type
+     * @param <V>    value type
+     * @param info   contains information about the assertion.
+     * @param actual the given {@code Map}.
+     * @param values the given values.
+     * @throws AssertionError       if the actual map is {@code null}.
+     * @throws AssertionError       if the actual map contains the given values.
+     * @throws NullPointerException if values vararg is {@code null}.
+     */
+    public <K, V> void assertDoesNotContainValues(AssertionInfo info, Map<K, V> actual,
+                                                  @SuppressWarnings("unchecked") V... values) {
+        assertNotNull(info, actual);
+        checkNotNull(values, "The array of values to look for should not be null");
+        if (actual.isEmpty() && values.length == 0) return;
```

Comment:
Why not `||`?

Reply:
Because if you have `actual` and expected `values` empty you can return with success. If `actual` would be empty but `values` not, then we have an error. The same for the opposite case.